ui/app/styles/components/ui-wizard.scss

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+// This file is not being used. https://github.com/hashicorp/vault/pull/19220
```

Comment:
Since this file isn't being used, should we remove the file?

Reply:
Good question. I asked Chelsea this and because we're not removing any of the UI wizard code, just not surfacing it, we're going to keep it. The idea is that in the future we'll revamp this component and so we don't want to remove anything.